### ISSUE TITLE
Suggestion: Filter/sort "Projects" page in Dashboard

### ISSUE THREAD

**Issue by tobiasvl** [itchio/itch.io#1050]
The "Projects" tab in the Dashboard (ie. the default tab) currently lists all my projects in alphabetical order. This is often not very useful. At this point I have quite a few projects, many of which are drafts, and it's cumbersome to scroll down to find the correct one with the orange "Draft" label. Sometimes I might even change the name of a draft game page during development, changing its place in the list.

It would be nice if I could filter or sort my projects by one or more of the following criteria, in addition to alphabetical order:

* Visibility status (Draft/Restricted/Published)
* Date (many possibilities; last update of game page, publication date if applicable, creation date regardless of publication status, etc)
* Popularity (based on some metric, views/downloads)
* Probably more

**Comment by ashmys** [comment]
Yes, please add this necessary feature to Creator Dashbord itch io.
its has been 6 years
